lib/ChainwebData/Types.hs

```diff
@@ -67,6 +69,10 @@ asBlock (PowHeader bh ph) m = Block
 hashToDbHash :: Hash -> DbHash t
 hashToDbHash = DbHash . hashB64U
 
+-- inverse of hashToDbHash
+dbHashToHash :: DbHash t -> Either String Hash
```

Comment:
This doesn't appear to be used anywhere in this PR.  In the absence of full-up testing to guarantee compatibility with the changes we just made to our base64 encoding, I think it would be better to leave this out.